lib/source-map.js

```diff
@@ -114,20 +114,23 @@ class SourceMap {
 
     let valid = true;
```

Comment:
This variable is not needed once you remove the validation.

Reply:
Done